cli/types/cypress.d.ts

```diff
@@ -625,6 +625,20 @@ declare namespace Cypress {
      */
     as(alias: string): Chainable<Subject>
 
+    /**
+     * Attach a file to the given <input> element, or drag and drop a file over any subject.
+     *
+     * @param {FileReference} files - The file(s) to attach or drag onto this element.
+     * @see https://on.cypress.io/TODO
```

Comment:
```suggestion
     * @see https://on.cypress.io/attachFile
```

Saw this link was used elsewhere, so figure it's the one that will be used here.